src/js/components/Feedback/Feedback.js

```diff
@@ -0,0 +1,140 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { ThemeContext } from 'styled-components';
+import { FormClose } from 'grommet-icons';
+import { PositionedFeedbackBox } from './StyledFeedbackButton';
+import { AnnounceContext } from '../../contexts/AnnounceContext';
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { Form } from '../Form';
+import { Heading } from '../Heading';
+import { Layer } from '../Layer';
+import { Text } from '../Text';
+
+const Announcer = ({ announce, message, mode, role }) => {
+  const theme = useContext(ThemeContext);
+  announce(message, mode);
+  return (
+    <Text role={role} aria-live={mode} {...theme?.feedback?.success}>
+      {message}
+    </Text>
+  );
+};
+
+const AnnounceContextComponent = (props) => (
+  <AnnounceContext.Consumer>
+    {(announce) => <Announcer announce={announce} {...props} />}
+  </AnnounceContext.Consumer>
+);
+
+export const Feedback = ({
+  children,
+  feedbackButton,
+  layerProps,
+  messages,
+  modal,
+  onSubmit,
+  show,
+  title,
+}) => {
+  const theme = useContext(ThemeContext);
+  // tracks if feedback has successfully been submitted
+  const [successfulSubmit, setSuccessfulSubmit] = useState(false);
+  const [open, setOpen] = useState(false);
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
+
+  // Want to show Thank you message so close the modal after 2 seconds
+  // should we change the timing to be in theme to have user control?
+  const closeFeedbackModal = () => {
+    setTimeout(() => {
+      setOpen(false);
+    }, 2000);
+  };
```

Comment:
Not sure if this rule would apply to the time limit set here https://www.w3.org/WAI/WCAG21/Understanding/timing-adjustable.html

Reply:
oh yeah I was going to see if we should change this or can always have it in the theme one of the questions I have for the research that was done